azure-ts-appservice-springboot/README.md

```diff
@@ -1,53 +1,58 @@
-[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
-
 # Deploy a Spring Boot App using Jenkins and Pulumi
 
-This example shows how you can deploy a Spring Boot app to an Azure App Service instance using Pulumi in a Jenkins Pipeline. The Spring Boot app is packaged into a container image that is automatically (and conveniently!) built as part of the Pulumi app. The container image is pushed up to a private Azure Container Registry and then used as the source for an App Service instance.
+This example shows how you can deploy a Spring Boot app to an Azure App Service instance using Pulumi in a Jenkins Pipeline. The Spring Boot app is packaged into a container image that is conveniently built as part of the Pulumi app. The container image is pushed up to a private Azure Container Registry and then used as the source for an App Service instance.
+
+## Prerequisites
+
+1.  [Install Pulumi](https://www.pulumi.com/docs/get-started/install)
+1.  [Configure Azure credentials](https://www.pulumi.com/docs/intro/cloud-providers/azure/setup)
 
-## Running the App
+## Steps
 
-1.  Create a new stack:
+### Step 1: Create a new stack
 
-    ```
-    $ pulumi stack init dev
-    ```
+```
+$ pulumi stack init dev
+```
 
-1.  Login to Azure CLI (you will be prompted to do this during deployment if you forget this step):
+### Step 2: Log in to the Azure CLI
 
-    ```
-    $ az login
-    ```
+You will be prompted to do this during deployment if you forget this step.
 
-1.  Restore NPM dependencies:
+```
+$ az login
+```
 
-    ```
-    $ cd infrastructure
-    $ npm install
-    ```
+### Step 3: Install NPM dependencies
 
-1.  Run `pulumi up` to preview and deploy changes:
+```
+$ cd infrastructure
```

Comment:
I think `$ cd infrastructure` should move up to be the first line in step 1 (before creating a stack).